index.bs

```diff
@@ -503,10 +564,10 @@ When the <dfn method for="XRSession">cancelAnimationFrame(|handle|)</dfn> method
     1. If an exception is thrown, [=report the exception=].
   1. Set |session|'s [=processing frame=] to <code>false</code>.
   1. Set |frame|'s [=active=] boolean to <code>false</code>.
+  1. If |session|'s [=list of pending render states=] is not empty, [=apply pending render states=].
```

Comment:
Hmm, I think this should probably be the first step so that each frame has the freshest data.